### DISCUSSION TITLE
When will version v3.4.0 be released?

### DISCUSSION THREAD

**Discussion by gada121982** [cubefs/cubefs#3296] [Q&A] [answered]
Hello team, I am waiting for version 3.4.0 to have the s3 bucket lifecycle feature. From what I see, it is currently in development. Can you estimate when this version will be released, and will version 3.4.0 contain all the fixes from the previous version? Please reply me soon.

Thank you team so much, have a nice day.

**Comment by leonrayang** [comment, marked as answer]
@gada121982 The release of version 3.4.0 is scheduled for June

**Reply by gada121982** [reply]
Thank you for letting me know, I'm really looking forward to this release.
